nailgun/entities.py

```diff
@@ -4383,7 +4383,8 @@ def download_debug_certificate(self, synchronous=True, **kwargs):
         kwargs.update(self._server_config.get_client_kwargs())
         response = client.get(
             self.path('download_debug_certificate'), **kwargs)
-        return _handle_response(response, self._server_config, synchronous)
+        return str(_handle_response(
+            response, self._server_config, synchronous))
```

Comment:
This will not work in py3 this will return something like b'***'. 
Please use decode('utf-8') that will work in py2 and py3.